ajax/libs/jquery-file-upload/package.json

```
@@ -0,0 +1,26 @@
+{
+  "name": "jquery-file-upload",
+  "filename": "jquery-file-upload.js",
+  "description": "jQuery Upload File plugin provides Multiple file Uploads with progress bar.Works with any server-side platform (Google App Engine, PHP, Python, Ruby on Rails, Java, etc.) that supports standard HTML form file uploads.",
+  "keywords": [
+    "jquery",
+    "upload",
+    "file",
+    "ajax",
+    "progressbar"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "git@github.com:hayageek/jquery-upload-file.git"
+  },
+  "license": "MIT",
+  "npmName": "jquery-file-upload",
+  "npmFileMap": [
+    {
+      "basePath": "dist",
```

`dist` directory is not present in the distributed `jquery-file-upload` package.

---

```suggestion
      "basePath": "js",
```

I would guess you probably want the `js` directory instead:

```
ls node_modules/jquery-file-upload/js

jquery.uploadfile.js     jquery.uploadfile.min.js
```